images/kernel/upgrade-config.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+KERNEL_BUILDER_IMAGE=luxas/kernel-builder:gcc-7-2
```

Comment:
This is not a publicly accessible image tag: https://hub.docker.com/r/luxas/kernel-builder/tags
Did you mean `gcc-7-2019-02-06` or `gcc-7` ?